nautobot/core/settings.py

```diff
@@ -690,7 +693,10 @@
 # django-rq settings will use the same instance/database by default.
 CACHES = {
     "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
+        "BACKEND": os.getenv(
+            "NAUTOBOT_CACHES_BACKEND",
+            "django_prometheus.cache.backends.redis.RedisCache" if METRICS_ENABLED else "django_redis.cache.RedisCache",
+        ),
```

Comment:
Can you update `nautobot/core/templates/nautobot_config.py.j2` to reflect the changes to this file?